docs/content/Examples-Tutorials-Recipes/Multiple-Data-Sources-With-Same-Schema.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Manage Access to Multiple Data Sources
+permalink: /recipes/multiple-sources-same-schema
+category: Examples & Tutorials
+subCategory: Data sources
+menuOrder: 3
+---
+
+## Use case
+
+We need to access the data from different data sources for different tenants.
+For example, we are the platform for the online website builder, and each client
+can only view their data. The same data schema is used for all clients.
+
+## Configuration
+
+Each client has own database. To manage access to the databases, we need to use
+the
+[`contextToAppId`](https://cube.dev/docs/config#options-reference-context-to-app-id)
+function and the
+[`driverFactory`](https://cube.dev/docs/config#options-reference-driver-factory)
+property. [JSON Web Token](https://cube.dev/docs/security) includes information
+about the tenant name in the `tenant` property.
```

Comment:
1. "To manage access" sounds quite vague. You're referencing two configuration properties, so it's great to explain what each of them does:

```
To enable multitenancy, use the `contextToAppId` function to provide distinct identifiers for each tenant.
Also, implement the `driverFactory` function where you can select a data source based on the tenant name.
```